agenda-webapps/src/main/webapp/vue-app/agenda/components/top-toolbar/AgendaSwitchView.vue

```diff
@@ -18,15 +18,15 @@
 <script>
 export default {
   props: {
-    calendarType: {
+    agendaPeriodType: {
```

Comment:
if this is a data managed by the component and not passed by parent Vue component, better change this to a `data` attribute.

Reply:
we need to get the value configured by the user at first.